aidial_sdk/header_propagator.py

```diff
@@ -23,7 +23,7 @@ def __init__(
     async def __call__(
         self, scope: Scope, receive: Receive, send: Send
     ) -> None:
-        for header in scope["headers"]:
+        for header in scope.get("headers") or []:
```

Comment:
You could set a default: `scope.get("headers", [])`

Reply:
This is a safer way because it will work in the following case:
```json
{
    "headers": null
}
```